### PR TITLE
Elden carpool creation

### PR DESCRIPTION
**Pull request by eldenyap48**
## Proof that your changes work:
- For frontend changes, include screenshots / videos of the UI
- For backend changes, include screenshots of successful Postman API calls

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have DM'd the group chat that I have made a PR and need reviewers.
- [ ] I haved moved the Trello issue to "Under Review"
- [ ] I have checked the Chrome console in Developer Tools to make sure there are no significant errors

### CODE REVIEW

File: calpool-be/app.py [hunk 1]
```diff
@@ -34,6 +35,37 @@ def nothing():
 def pingpong():
     return "pong"
 
+@app.route('/create_trip', methods=['POST'])
+def create_carpool():
```
Comment (DylanHuynh) [inline calpool-be/app.py]: rename to make same name as the route above

File: calpool-be/app.py [hunk 2]
```diff
@@ -34,6 +35,37 @@ def nothing():
...
+        end_time= datetime.strptime(end_time, '%m/%d/%y %H:%M'),
+        lower_bound=lower_bound,
+        upper_bound=upper_bound,
+        max_people=max_people,
+        comments=comments,
+        owner=current_user)
+
+    new_trip.save()
+
+    return "done"
```
Comment (DylanHuynh) [inline calpool-be/app.py]: I think it'd be useful to return the Trip id to be a little more descriptive instead. you can follow what lucas did for user id

**Comment by eldenyap48** [comment]
just changed

**Comment by eldenyap48** [comment]
just made other changes

**Pull request merged**